k8s-install/scripts/install-cni.sh

```diff
@@ -125,6 +125,7 @@ sed -i s~__ETCD_CERT_FILE__~${CNI_CONF_ETCD_CERT:-}~g $TMP_CONF
 sed -i s~__ETCD_KEY_FILE__~${CNI_CONF_ETCD_KEY:-}~g $TMP_CONF
 sed -i s~__ETCD_CA_CERT_FILE__~${CNI_CONF_ETCD_CA:-}~g $TMP_CONF
 sed -i s~__ETCD_ENDPOINTS__~${ETCD_ENDPOINTS:-}~g $TMP_CONF
+sed -i s~__LOG_LEVEL__~${LOG_LEVEL}~g $TMP_CONF
```

Comment:
I think the defaulting should be done here rather than in the Dockerfile

e.g.

```
${LOG_LEVEL:-warn}
```